pkg/telemetry/telemetry_noop.go

```diff
@@ -0,0 +1,15 @@
+//go:build !telemetry
+// +build !telemetry
+
+package telemetry
+
+import (
+	"context"
+
+	"github.com/rs/zerolog/log"
+)
+
+func Start(ctx context.Context, honeycomb_api_key string, honeycomb_dataset string) {
+	log.Info().Msg("NO telemetry here..")
```

Comment:
I think it might be safer to remove this log message.

Reply:
I think we can remove the log message and there's no trouble adding a test dataset and providing that as the environment variables during test with the "prod" dataset being specified during release builds!

We could maybe look later to add some telemetry to the tests themselves then, similar to how litmus does, and show trends/failures. 🤔